test/e2e/reconnecting.feature

```diff
@@ -12,6 +12,11 @@ Feature: Passing Options
         'karma-jasmine',
         'karma-chrome-launcher'
       ];
+      client = {
+        jasmine: {
+          random: false
```

Comment:
So Jasmine 3 executes tests in random order by default and since this particular spec relies on the tests being executed sequentially I have disabled the random execution feature.